.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @codacy/toss
```

Comment:
This is crazy good! No need to change all repositories when the team changes!

Reply:
Yeah! We should do it for the rest of the repositories 👌